### PR TITLE
Identity | Bump identity lib to add new consent copy

### PR DESCRIPTION
**Pull request by alexduf**
## What does this change?
The new version of the identity model contains new copy for the communication consents, this will update it.

Edit @coldlink:
This also hides the new consents that were previously added in on the old consents flow page at /consents/thank-you

## Does this change need to be reproduced in dotcom-rendering ?

- [x] No
- [ ] Yes (please indicate your plans for DCR Implementation)

### Does this affect other platforms?

- [ ] AMP 
- [ ] Apps
- [ ] Other (please specify)

### Does this affect GLabs Paid Content Pages? Should it have support for Paid Content?

- [x] No
- [ ] Yes (please give details)

### Does this change break ad-free?

- [x] No
- [ ] It did, but tests caught it and I fixed it
- [ ] It did, but there was no test coverage so I added that then fixed it

### Does this change update the version of CAPI we're using?

- [x] No, all the existing database files are just fine
- [ ] Yes, and I have [re-run all the tests locally and checked in all the updated data/database/xyz files](https://github.com/guardian/frontend/blob/main/docs/03-dev-howtos/15-updating-test-database.md)

### Accessibility test checklist

- [ ] [Tested with screen reader](https://accessibility.gutools.co.uk/testing/web/screen-readers/)
- [ ] [Navigable with keyboard](https://accessibility.gutools.co.uk/testing/web/keyboard-navigation/)
- [ ] [Colour contrast passed](https://accessibility.gutools.co.uk/testing/web/colour-contrast/)

### Tested

- [x] Locally
- [x] On CODE (optional)

### CODE REVIEW

**Comment by alexduf** [comment]
That seems to trigger a library conflict, so don't merge while I investigate 🤔

**Comment by coldlink** [comment]
fixed by publishing a branch of the identity library in https://github.com/guardian/identity/pull/1875

File: identity/app/views/support/fragment/ConsentChannel.scala [hunk 1]
```diff
@@ -1,15 +1,19 @@
...
-  case object ProfilingConsentChannel extends ConsentChannelBehaviour("profiling_optout")
+  // channels
+  case object TextConsentChannel extends ConsentChannelBehaviour(Consent.CommunicationSms.id)
+  case object PhoneOptOutConsentChannel extends ConsentChannelBehaviour(Consent.PhoneOptout.id)
+  case object PostOptOutConsentChannel extends ConsentChannelBehaviour(Consent.PostOptout.id)
+  case object MarketResearchConsentChannel extends ConsentChannelBehaviour(Consent.MarketResearchOptout.id)
+  case object ProfilingConsentChannel extends ConsentChannelBehaviour(Consent.ProfilingOptout.id)
+
+  // products
+  // case object TextConsentChannel extends ConsentChannelBehaviour(Consent.Events.id)
```
Comment (dominickendrick) [inline identity/app/views/support/fragment/ConsentChannel.scala:16]: Does this line need to stay ?

**Review by dominickendrick — APPROVED** [review]
Looks good !

File: identity/app/views/support/fragment/ConsentChannel.scala (hunk 1 above)
Comment (coldlink) [reply identity/app/views/support/fragment/ConsentChannel.scala]: ah ha! nope, have removed

**Pull request merged**